### PR TITLE
feat: Add injected type Document to protobufs

### PR DESCRIPTION
**Pull request by mykola-vrmchk**

### CODE REVIEW

**Comment by mykola-vrmchk** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2632** <a href="https://app.graphite.dev/github/pr/dxos/dxos/2632" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dxos/dxos/2632?utm_source=stack-comment).

**Review by dmaretskyi — APPROVED** [review]

File: packages/experimental/kai/src/frames/Stack/StackContent.tsx [hunk 1]
```diff
@@ -23,10 +23,10 @@ export const StackContent: FC<{ config: Config; space: Space; object: EchoObject
 }) => {
   // TODO(burdon): Type?
   switch ((object as any).__typename) {
```
Comment (richburdon) [inline packages/experimental/kai/src/frames/Stack/StackContent.tsx:25]: we can remove this now?

File: packages/experimental/kai/src/frames/Stack/StackContent.tsx [hunk 2]
```diff
@@ -23,10 +23,10 @@ export const StackContent: FC<{ config: Config; space: Space; object: EchoObject
...
-    case DocumentType.type.name: {
+    case TextDocument.type.name: {
       return (
         <Composer
-          document={(object as DocumentType).content}
+          document={(object as TextDocument).content}
```
Comment (richburdon) [inline packages/experimental/kai/src/frames/Stack/StackContent.tsx:29]: Rename back to Document (with `as` in the import)

File: packages/experimental/kai/src/proto/schema.proto [hunk 3]
```diff
@@ -119,13 +119,15 @@ message DocumentStack {
   option (object) = true;
 
   message Section {
+    option (object) = true;
+
     string object_id = 1;
-    Document document = 10; // TODO(burdon): Recursion error.
+    dxos.schema.Document document = 10;
     File file = 11;
```
Comment (richburdon) [inline packages/experimental/kai/src/proto/schema.proto:126]: Remove File.

**Pull request merged**